test/ui/LoginSystemTestsPage.ts

```diff
@@ -0,0 +1,55 @@
+import { WebElement, By, until } from 'selenium-webdriver';
+import { expect } from 'chai';
+import { basepage } from '../ui/basepage';
+
+export class LoginTestsPage extends basepage { 
+
+    // Web Elements --------------------------------------------------------------------------------
+
+    public static async HomePageImg(): Promise<WebElement> {
+        return this.driver.findElement(By.xpath("//img[@id='homepageBackgroundImage']"));
```

Comment:
can we not get this by id rather than xpath?

Reply:
completed this change